src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -1,22 +1,38 @@
 package mate.academy.bookstore.model;
 
+import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Data
 @Entity
+@Table(name = "books")
+@NoArgsConstructor
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
 public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @Column(nullable = false)
     private String title;
+    @Column(nullable = false)
     private String author;
+    @Column(nullable = false, unique = true)
     private String isbn;
+    @Column(nullable = false)
     private BigDecimal price;
     private String description;
+    @Column(name = "cover_image")
```

Comment:
```suggestion
```

Hibernate automatically creates column name in snake_case style, so don't need to specify it